src/resolve-config.ts

```diff
@@ -88,3 +90,15 @@ export const getDefinedInput = (name: string): string | undefined => {
 
   return input !== '' ? input : undefined
 }
+
+export const getDefinedBoolean = (name: string): boolean | undefined => {
+  try {
+    if (!getDefinedInput(name)) {
+      return undefined
+    }
+    return core.getBooleanInput(name)
+  } catch (error) {
+    core.setFailed(`'${name}' configuration value does not meet YAML 1.2 specification for Boolean type`)
```

Comment:
I think we should keep the original error message, as it also contains hints on the correct format (i.e. "true|false, True|False..."), and if I remember correctly, the original `error.message` also contains the input name "tunnel"